pageserver/src/tenant/timeline.rs

```diff
@@ -1252,7 +1268,7 @@ impl Timeline {
                     &timeline_id,
                     crate::metrics::EvictionsWithLowResidenceDurationBuilder::new(
                         "mtime",
-                        conf.evictions_low_residence_duration_metric_threshold,
+                        evictions_low_residence_duration_metric_threshold,
```

Comment:
We need to update the threshold stored inside the metric when it gets updated via mgmt API.
Should have a test for this as well.